src/drop-down/drop-down-item.component.ts

```diff
@@ -0,0 +1,167 @@
+import {
+    Component,
+    ElementRef,
+    forwardRef,
+    HostBinding,
+    HostListener,
+    Inject,
+    Input } from "@angular/core";
+import { IgxDropDownComponent, ISelectionEventArgs } from "./drop-down.component";
+
+/**
+ * The `<igx-drop-down-item> is a container intended for row items in
+ * a `<igx-drop-down>` container.
+ */
+@Component({
+    selector: "igx-drop-down-item",
+    templateUrl: "drop-down-item.component.html"
+})
+
+export class IgxDropDownItemComponent {
+    private _isFocused = false;
+
+    @HostBinding("class.igx-drop-down__item")
+    get itemStyle(): boolean {
+        return !this.isHeader;
+    }
+
+    /**
+     * Gets if the given item is selected
+     */
+    get isSelected() {
```

Comment:
That one is not clear at all. Which is the given item? we should either rename the getter, or update the description.

Reply:
Done